api/src/main/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimController.java

```diff
@@ -52,13 +51,8 @@
     @ApiResponses({@ApiResponse(code = 400, message = "Bad request", response = ErrorResponse.class)})
     public ResponseEntity<ClaimResultDTO> createClaim(@RequestBody @Valid @ApiParam("The claim to persist") NewClaimDTO newClaimDTO) {
         log.debug("Received claim");
-        Claimant claimant = claimantConverter.convert(newClaimDTO.getClaimant());
```

Comment:
Whilst I agree with this change, it seems unrelated to this story and in future should be done in it's own PR.

Reply:
Fair point. We did it because we were at the point of creating a new converted for the EligibilityOverride, and this felt cleaner. Should have been in a separate pr though.